src/test/resources/raw-jsonschema/beer-schema.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```

Comment:
why was this removed?

Reply:
It's a link to json-schema draft v7, I've replaced it with `http://json-schema.org/draft-04/schema#` now.